src/fonduer/parser/spacy_parser.py

```diff
@@ -94,7 +51,25 @@ class Spacy(object):
     def __init__(self, lang):
         self.logger = logging.getLogger(__name__)
         self.name = "spacy"
-        self.model = Spacy.load_lang_model(lang)
+        self.spacy_languages = ["en", "de", "es", "pt", "fr", "it", "nl", "xx"]
+        self.alpha_languages = {"ja": "Japanese"}
```

Comment:
`spacy_languages` and `alpha_languages` are a bit confusing because "spacy" and "alpha" are not counterparts.
Can you make it `languages` and `alpha_languages`, or `supported_languages` and `alpha_supported_languages`, or something like these?